CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.2.0
+
+### Fixes
+- Additional properties have been added to `ProblemDetails` to more accurately represent the API response.
+- **[BREAKING]** Remove `ProblemDetails` inherit from `HashMap<String, Object>`. Undocumented properties can be retrieved using `ProblemDetails.getExtensions()` instead.
+- **[BREAKING]** Remove `ApiException` `response` property. Should use `ApiException` `message` property instead. Additionally, the `ApiException` constructor has been changed to not require the `response` property.
+
```

Comment:
im not sure if this is necessary or not, but should you also indicate that tokenClients have also been updated?

Reply:
ok i mentioned improved token client error handling